File: test/fixtures/output/kotlin/okhttp/full.kt
val client = OkHttpClient()

val mediaType = MediaType.parse("application/x-www-form-urlencoded")
val body = RequestBody.create(mediaType, "foo=bar")
val request = Request.Builder()
  .url("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")
  .post(body)
  .addHeader("cookie", "foo=bar; bar=baz")
  .addHeader("accept", "application/json")
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .build()

val response = client.newCall(request).execute()

File: __tests__/__fixtures__/output/node/request/full.js
const request = require('request');

const jar = request.jar();
jar.setCookie(request.cookie('foo=bar'), 'https://httpbin.org/anything');
jar.setCookie(request.cookie('bar=baz'), 'https://httpbin.org/anything');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value',
  headers: {
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  form: {foo: 'bar'},
  jar: jar
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

